src/models/TechRecordsDAO.ts

```diff
@@ -181,6 +185,15 @@ class TechRecordsDAO {
           resource: "/system-number/",
         })
 
+  public getZNumber = () =>
```

Comment:
I'm not sure if there are some unit tests for the trailiD which can be copied over